lexicons/app/bsky/purchase/getSubscriptions.json

```diff
@@ -0,0 +1,41 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.purchase.getSubscriptions",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Enumerate subscriptions for the requesting account, and the email used in the purchase. Requires auth.",
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["subscriptions"],
+          "properties": {
+            "email": { "type": "string" },
+            "subscriptions": {
+              "type": "array",
+              "items": {
+                "type": "ref",
+                "ref": "#subscription"
+              }
+            }
+          }
+        }
+      }
+    },
+    "subscription": {
+      "type": "object",
+      "required": [],
+      "properties": {
+        "status": { "type": "string" },
+        "renewalStatus": { "type": "string" },
```

Comment:
Nit: could add some `knownValues` fields here so we get more strict types

Reply:
yup you can 👍 
it's basically a documentation mechanism for string fields that says what the expected values are. but you can always put any string in it